pages/instructorAssessmentDownloads/instructorAssessmentDownloads.sql

```diff
@@ -82,7 +82,8 @@ SELECT DISTINCT ON (ai.id, q.qid)
     s.id AS submission_id,
     v.params,
     v.true_answer,
-    (s.submitted_answer - '_files') AS submitted_answer
+    (s.submitted_answer - '_files') AS submitted_answer,
+    s.partial_scores
```

Comment:
Should this also be added to the API for the assessment export?

Reply:
Yeah, might as well.  I'll add that.